packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -1334,6 +1334,12 @@ export function resolveAvailableTypes(
         .filter(type => {
             // Can always return to the original type for field
             if (type.name === field.dataType.name) return true;
+            if ((type.rangeURI === STRING_RANGE_URI || type.rangeURI === MULTILINE_RANGE_URI) && !type.conceptURI) return true;
```

Comment:
I think having this logic be encapsulated in `acceptablePropertyType` could better match the logical pattern here.

Reply:
I could be incorrect about this -- just something I noticed at a quick glance. Definitely doesn't block merging.

Reply:
I added comments related to this but recommend keeping it as-is. acceptablePropertyType() is currently concerned about range URI and concept URI matching for a single type (like knowing that Text and Subject are the same, and Integer and User are the same).